crates/nu-cli/tests/folder_completions.rs

```diff
@@ -27,3 +29,27 @@ fn folder_completions() {
     // Match the results
     match_suggestions(expected_paths, suggestions);
 }
+
+#[test]
+fn folder_completions_with_dots() {
+    // Create a new engine
+    let (dir, dir_str, engine, stack) = new_engine();
+
+    // Instatiate a new completer
+    let mut completer = NuCompleter::new(std::sync::Arc::new(engine), stack);
+
+    // Test completions for the current folder
+    let parent_dir = PathBuf::from(dir_str).join("..");
+    let target_dir = format!("cd {}", parent_dir.to_str().unwrap());
+    let suggestions = completer.complete(&target_dir, target_dir.len());
+
+    // Create the expected values
+    let expected_paths: Vec<String> = vec![
+        folder(dir.join("..").join("formats")),
+        folder(dir.join("..").join("playground")),
+        folder(dir.join("..").join("completions")),
+    ];
```

Comment:
Can you also add a test case for triggering the completions on a single dot hidden folder or in a case where no initial path is present?

Reply:
Our text fixture should probably also include directories with unicode names.